src/server_manager/ui_components/outline-server-settings.html

```diff
@@ -208,6 +240,76 @@ <h3>[[localize('settings-metrics-header')]]</h3>
           this.fire(metricsSignal);
         }
       },
+      _keysPortButtonsVisible: function(visible) {
+        this.$.keysPortSaveButton.hidden = !visible;
+        this.$.keysPortCancelButton.hidden = !visible;
+      },
+      _keysPortButtonsEnabled: function(enabled) {
+        this.$.keysPortSaveButton.disabled = !enabled;
+        this.$.keysPortCancelButton.disabled = !enabled;
+      },
+      _handleKeysPortClick: function() {
+        this._keysPortButtonsVisible(true);
+        const invalid = this.$.keysPortInput.invalid;
+        this.$.keysPortSaveButton.disabled = invalid;
+        this.$.keysPortCancelButton.disabled = false;
+      },
+      _handleKeysPortServerSuccess() {
+        console.info('Port successfully changed');
```

Comment:
We generally avoid logging in HTML. We should be able to tell whether the port change succeded form the API request logs.